src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,168 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev1 (02/24/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Upgrade to Pex 2.1.4 (#8881)
+  `PR #8881 <https://github.com/pantsbuild/pants/pull/8881>`_
```

Comment:
I'm wondering if we should mention this in the email, given that multiple users have expressed how important this is to them? Something like:

> This release increases several substantial improvements, including using Pex 2.x which means that Pants now properly supports `manylinux2014` wheels. See ... for the release notes.

Thoughts?

Reply:
Yes, worth calling out.